homeassistant/components/camera/synology.py

```diff
@@ -58,6 +58,24 @@
 
     cameras = surveillance.get_all_cameras()
 
+    
+    def handle_enable_home_mode(call):
+    """Register a service to be able to enable home mode."""
+        surveillance.set_home_mode(True)
+
+    hass.services.async_register(DOMAIN,
```

Comment:
Custom services should be registered under the integrating component domain, ie synology.